test/unit/get-commit-test.js

```diff
@@ -0,0 +1,51 @@
+const simple = require('simple-mock')
+const {test} = require('tap')
+
+const getCommit = require('../../lib/get-commit')
+
+const api = {
+  repos: {
+    getCommit: () => {}
+  }
+}
+
+test('get commit request succeeds', t => {
+  const state = {
+    api,
+    debug: () => {},
+    owner: 'owner',
+    repo: 'repo',
+    sha: 'sha'
+  }
+
+  simple.mock(api.repos,
+  'getCommit').resolveWith({
```

Comment:
why the line break here?

Reply:
oops. i was working with split panes and instead of using line wrap I pressed enter for a new line. weird.